[2025-04-15T11:10:47.683+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-15T11:10:47.807+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.run_dbt_trusted scheduled__2025-04-15T07:53:00+00:00 [queued]>
[2025-04-15T11:10:47.874+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.run_dbt_trusted scheduled__2025-04-15T07:53:00+00:00 [queued]>
[2025-04-15T11:10:47.877+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-04-15T11:10:47.978+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): run_dbt_trusted> on 2025-04-15 07:53:00+00:00
[2025-04-15T11:10:48.015+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=4069) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-15T11:10:48.026+0000] {standard_task_runner.py:72} INFO - Started process 4071 to run task
[2025-04-15T11:10:48.026+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'extract_raw_data_pipeline', 'run_dbt_trusted', 'scheduled__2025-04-15T07:53:00+00:00', '--job-id', '114', '--raw', '--subdir', 'DAGS_FOLDER/lead_data_pipeline.py', '--cfg-path', '/tmp/tmpvxeonbi2']
[2025-04-15T11:10:48.036+0000] {standard_task_runner.py:105} INFO - Job 114: Subtask run_dbt_trusted
[2025-04-15T11:10:48.395+0000] {task_command.py:467} INFO - Running <TaskInstance: extract_raw_data_pipeline.run_dbt_trusted scheduled__2025-04-15T07:53:00+00:00 [running]> on host c0ac39583bd8
[2025-04-15T11:10:48.927+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_raw_data_pipeline' AIRFLOW_CTX_TASK_ID='run_dbt_trusted' AIRFLOW_CTX_EXECUTION_DATE='2025-04-15T07:53:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-15T07:53:00+00:00'
[2025-04-15T11:10:48.932+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-15T11:10:49.046+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-04-15T11:10:49.052+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'set -x; cd /opt/***/dbt && dbt run --select tag:trusted']
[2025-04-15T11:10:49.095+0000] {subprocess.py:86} INFO - Output:
[2025-04-15T11:10:49.107+0000] {subprocess.py:93} INFO - + cd /opt/***/dbt
[2025-04-15T11:10:49.111+0000] {subprocess.py:93} INFO - + dbt run --select tag:trusted
[2025-04-15T11:26:21.104+0000] {job.py:229} INFO - Heartbeat recovered after 927.90 seconds
[2025-04-15T11:26:23.571+0000] {subprocess.py:93} INFO - [0m11:26:23  Running with dbt=1.8.0
[2025-04-15T11:26:24.022+0000] {subprocess.py:93} INFO - [0m11:26:24  Registered adapter: postgres=1.8.2
[2025-04-15T11:26:24.650+0000] {subprocess.py:93} INFO - [0m11:26:24  Found 9 models, 10 sources, 428 macros
[2025-04-15T11:26:24.660+0000] {subprocess.py:93} INFO - [0m11:26:24
[2025-04-15T11:26:24.856+0000] {subprocess.py:93} INFO - [0m11:26:24  Concurrency: 1 threads (target='dev')
[2025-04-15T11:26:24.857+0000] {subprocess.py:93} INFO - [0m11:26:24
[2025-04-15T11:26:24.873+0000] {subprocess.py:93} INFO - [0m11:26:24  1 of 4 START sql table model lead_trusted.non_pii_data ......................... [RUN]
[2025-04-15T11:42:32.777+0000] {job.py:229} INFO - Heartbeat recovered after 929.96 seconds
[2025-04-15T11:58:46.952+0000] {job.py:229} INFO - Heartbeat recovered after 927.81 seconds
[2025-04-15T12:11:30.864+0000] {job.py:229} INFO - Heartbeat recovered after 722.80 seconds
[2025-04-15T12:27:43.245+0000] {job.py:229} INFO - Heartbeat recovered after 931.22 seconds
[2025-04-15T12:43:55.691+0000] {job.py:229} INFO - Heartbeat recovered after 931.12 seconds
[2025-04-15T12:44:03.280+0000] {subprocess.py:93} INFO - [0m12:44:03  1 of 4 OK created sql table model lead_trusted.non_pii_data .................... [[32mSELECT 21225910[0m in 4658.40s]
[2025-04-15T12:44:03.284+0000] {subprocess.py:93} INFO - [0m12:44:03  2 of 4 START sql table model lead_trusted.payment_data ......................... [RUN]
[2025-04-15T12:59:47.786+0000] {job.py:229} INFO - Heartbeat recovered after 904.76 seconds
[2025-04-15T13:12:33.096+0000] {job.py:229} INFO - Heartbeat recovered after 743.89 seconds
[2025-04-15T13:28:23.745+0000] {job.py:229} INFO - Heartbeat recovered after 930.09 seconds
[2025-04-15T13:28:30.893+0000] {subprocess.py:93} INFO - [0m13:28:30  2 of 4 OK created sql table model lead_trusted.payment_data .................... [[32mSELECT 3977366[0m in 2667.61s]
[2025-04-15T13:28:30.896+0000] {subprocess.py:93} INFO - [0m13:28:30  3 of 4 START sql table model lead_trusted.pii_data ............................. [RUN]
[2025-04-15T14:13:34.028+0000] {job.py:229} INFO - Heartbeat recovered after 2689.38 seconds
[2025-04-15T14:29:29.078+0000] {job.py:229} INFO - Heartbeat recovered after 929.19 seconds
[2025-04-15T14:45:19.556+0000] {job.py:229} INFO - Heartbeat recovered after 904.05 seconds
[2025-04-15T15:00:46.716+0000] {job.py:229} INFO - Heartbeat recovered after 906.64 seconds
[2025-04-15T15:14:35.867+0000] {job.py:229} INFO - Heartbeat recovered after 808.59 seconds
[2025-04-15T15:30:24.566+0000] {job.py:229} INFO - Heartbeat recovered after 907.40 seconds
[2025-04-15T15:46:14.402+0000] {job.py:229} INFO - Heartbeat recovered after 929.25 seconds
[2025-04-15T15:50:30.843+0000] {job.py:229} INFO - Heartbeat recovered after 215.20 seconds
[2025-04-15T16:06:08.002+0000] {job.py:229} INFO - Heartbeat recovered after 932.14 seconds
[2025-04-15T16:11:53.820+0000] {job.py:229} INFO - Heartbeat recovered after 304.58 seconds
[2025-04-15T16:15:36.943+0000] {job.py:229} INFO - Heartbeat recovered after 181.79 seconds
[2025-04-15T16:31:13.807+0000] {job.py:229} INFO - Heartbeat recovered after 931.84 seconds
[2025-04-15T16:31:18.991+0000] {subprocess.py:93} INFO - [0m16:31:18  3 of 4 OK created sql table model lead_trusted.pii_data ........................ [[32mSELECT 21225910[0m in 10968.09s]
[2025-04-15T16:31:18.994+0000] {subprocess.py:93} INFO - [0m16:31:18  4 of 4 START sql table model lead_trusted.technical_data ....................... [RUN]
[2025-04-15T16:32:10.474+0000] {subprocess.py:93} INFO - [0m16:32:10  4 of 4 OK created sql table model lead_trusted.technical_data .................. [[32mSELECT 3154094[0m in 51.48s]
[2025-04-15T16:32:10.490+0000] {subprocess.py:93} INFO - [0m16:32:10
[2025-04-15T16:32:10.491+0000] {subprocess.py:93} INFO - [0m16:32:10  Finished running 4 table models in 5 hours 5 minutes and 45.83 seconds (18345.83s).
[2025-04-15T16:32:10.553+0000] {subprocess.py:93} INFO - [0m16:32:10
[2025-04-15T16:32:10.554+0000] {subprocess.py:93} INFO - [0m16:32:10  [32mCompleted successfully[0m
[2025-04-15T16:32:10.555+0000] {subprocess.py:93} INFO - [0m16:32:10
[2025-04-15T16:32:10.556+0000] {subprocess.py:93} INFO - [0m16:32:10  Done. PASS=4 WARN=0 ERROR=0 SKIP=0 TOTAL=4
[2025-04-15T16:32:11.421+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-04-15T16:32:11.448+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-15T16:32:11.448+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=extract_raw_data_pipeline, task_id=run_dbt_trusted, run_id=scheduled__2025-04-15T07:53:00+00:00, execution_date=20250415T075300, start_date=20250415T111047, end_date=20250415T163211
[2025-04-15T16:32:11.617+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-15T16:32:11.632+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-15T16:32:11.757+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
